generator/wrapper_gen.py

```diff
@@ -894,7 +894,22 @@ def compute(self,
         {{input_args|fmt('{}', 'sphinx', sep='\n')|indent(8)}}
         :rtype: {{cytype}}
         '''
-        return self._compute({{input_args|fmt('{}', 'name', sep=', ')}}, False)
+
+        import sys
+        if 'dppy' in sys.modules:
+            from dppy import runtime as rt
+            from _oneapi import set_queue_to_daal_context, release_queue
+
+            ctx = rt.get_current_context()
+            q = ctx.get_sycl_queue()
+            set_queue_to_daal_context(q)
+
+            res = self._compute({{input_args|fmt('{}', 'name', sep=', ')}}, False)
+
+            release_queue(q)
```

Comment:
@michael-smirnov Attempt to work with dppy context fails with segmentation fault. Looks like that we should not release the queue after computation or we do it incorrect at this moment. Please clarify that we are not trying to destroy the original dppy queue.

Reply:
@Alexander-Makaryev thank you! That is not an issue with the wrong destruction of queue, but it is a known issue caused by the undefined destruction of static environment objects of oneDAL and SYCL. We are keeping a queue inside oneDAL even when dppy is trying to release SYCL context - that is the root cause of that seqfault. 
Now I emulate a "stateless" approach when oneDAL forgets a queue after each operation. It solves the problem. In the future, we will implement a stateless approach natively in new oneDAL interfaces.